### ISSUE TITLE
pyenv: python: command not found 

### ISSUE THREAD

**Issue by junjielizero** [pyenv/pyenv#870]
## Env:

- Mac OSX :10.12.3
- pyenv: brew install pyenv
- Using Zsh: 
```
export PYENV_ROOT="$HOME/.pyenv"
export PATH="$PYENV_ROOT/bin:$PATH"
eval "$(pyenv init -)"
```

## Problem:
```
➜  ~ pyenv versions
* system (set by /Users/blackjett/.python-version)
  2.7.11
  3.5.2
➜  ~ pyenv local 3.5.2
➜  ~ pyenv versions
  system
  2.7.11
* 3.5.2 (set by /Users/blackjett/.python-version)
➜  ~ python
pyenv: python: command not found

The `python' command exists in these Python versions:
  2.7.11
```

It's OK when changing to `system` and `2.7.11`(pyenv install)

```
➜  ~ pyenv version
2.7.11 (set by /Users/blackjett/.python-version)
➜  ~ python
Python 2.7.11 (default, Mar 22 2017, 08:37:48)
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.42.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>>
```

## Analyze

- I may delete some related files which leads to this result.

## Trying 

- I had uninstall pyenv (`brew uninstall pyenv`) and install again.

## Hope

- Fixed the problem.

**Comment by blueyed** [comment]
> I may delete some related files which leads to this result.

Remove and reinstall 3.5.2 then?!

**Comment by junjielizero** [comment]
@blueyed The problem is that I didn't remove `pyenv`  completely. And it works when reinstalling again.

**Comment by blueyed** [comment]
@junjielizero 
So it is fixed then?  Or am I missing something?
Is there anything pyenv could do when you "delete some related files which leads to this result"?

**Comment by junjielizero** [comment]
It's fixed and find no problem now. I think  "delete some related files which leads to this result"  mislead you and it's my fault.  I mean that I carelessly delete some files under root directory(`rm -rf ~`). I guess this command makes some files about `pyenv` disappear, so I get the problem as above.

**Comment by llazzaro** [comment]
I had the same issue, if I execute python I got "python: command not found".
 I solved it with: *python local 2.7.13* (or whatever version you want).
